plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -331,12 +331,7 @@ public boolean configure(String name, Map<String, Object> params) throws Configu
             _storage.configure("StorageLayer", params);
         }
 
-        value = _configDao.getValue(Config.VmwareCreateFullClone.key());
-        if (value == null) {
-            _fullCloneFlag = false;
-        } else {
-            _fullCloneFlag = Boolean.parseBoolean(value);
-        }
+        _fullCloneFlag = StorageManager.VmwareCreateCloneFull.value();
```

Comment:
@DaanHoogland 
is it possible `_fullCloneFlag` is null ?

Reply:
Yes, create full clone has a default value.